Resources/Prototypes/borg_types.yml

```
@@ -80,9 +80,10 @@
     - BorgModuleCargo
 
   defaultModules:
-  - BorgModuleGrapplingGun
+  - BorgModuleTool
+  - BorgModuleTraversal
   - BorgModuleMining
-  - BorgModuleAppraisal
+  - BorgModuleSupply
```

Silly me, I forgot to change this in my last commit.
```suggestion
  - BorgModuleAppraisal
```